reconcile/aus/metrics.py

```diff
@@ -108,3 +108,15 @@ def __init__(self, integration: str, org_id: str, ocm_env: str) -> None:
                 org_id=org_id,
             ),
         )
+
+
+class AUSOrganizationVersionDataGauge(AUSBaseMetric, GaugeMetric):
```

Comment:
Why is this a Gauge and not a Counter? Do we expect soak days can go down?

Reply:
It's a value that comes from the state, so it's also not "counted".